packages/@aws-cdk/cloudformation-include/test/test-templates/outputs-with-references.json

```diff
@@ -0,0 +1,42 @@
+{
+  "Conditions": {
+    "AlwaysFalseCond": {
+      "Fn::Equals": [
+        {
+          "Ref": "AWS::Region"
+        },
+        "completely-made-up-region"
+      ]
+    }
+  },
+  "Parameters": {
+    "MyParam": {
+      "Type": "String"
+    }
+  },
+  "Resources": {
+    "Bucket": {
+      "Type": "AWS::S3::Bucket",
+      "Properties": {
+        "BucketName": "my-referenced-bucket-name"
+      },
+      "Condition": "AlwaysFalseCond"
+    }
+  },
+  "Outputs": {
+    "Output1": {
+      "Value": { "Ref": "Bucket" },
+      "Description": { "Ref": "MyParam" },
+      "Condition": "AlwaysFalseCond"
+    },
+    "Output2": {
```

Comment:
Not sure why we need 2 Outputs here... how about just one, and the value is something like

```json
"Value": {
  "Fn::Join": ["", [
    { "Ref": "MyParam" },
    { "Fn::GetAtt": "Bucket" }
  ]],
}
```